packages/dune-configurator/dune-configurator.2.7.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis: "Helper library for gathering system configuration"
+description: """
+dune-configurator is a small library that helps writing OCaml scripts that
+test features available on the system, in order to generate config.h
+files for instance.
+Among other things, dune-configurator allows one to:
+- test if a C program compiles
+- query pkg-config
+- import #define from OCaml header files
+- generate config.h file
+"""
+maintainer: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml/dune"
+doc: "https://dune.readthedocs.io/"
+bug-reports: "https://github.com/ocaml/dune/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
```

Comment:
It seems like dune-configurator now requires at least OCaml 4.06:
```
#=== ERROR while compiling dune-configurator.2.7.1 ============================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/ocaml/dune/releases/download/2.7.1/dune-2.7.1.tbz)
# path                 ~/.opam/4.04/.opam-switch/build/dune-configurator.2.7.1
# command              ~/.opam/4.04/bin/dune build -p dune-configurator -j 72 @install
# exit-code            1
# env-file             ~/.opam/log/dune-configurator-23-9dee53.env
# output-file          ~/.opam/log/dune-configurator-23-9dee53.out
### output ###
#       ocamlc otherlibs/configurator/src/.configurator.objs/byte/configurator__V1.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -safe-string -g -bin-annot -I otherlibs/configurator/src/.configurator.objs/byte -I otherlibs/configurator/src/.configurator.objs/public_cmi -I /home/opam/.opam/4.04/lib/csexp -I /home/opam/.opam/4.04/lib/result -intf-suffix .ml -no-alias-deps -open Configurator__ -o otherlibs/configurator/src/.configurator.objs/byte/configurator__V1.cmo -c -impl otherlibs/configurator/src/v1.ml)
# File "otherlibs/configurator/src/v1.ml", line 532, characters 13-27:
# Error: Unbound value Int.Map.update
#     ocamlopt otherlibs/configurator/src/.configurator.objs/native/configurator__V1.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlopt.opt -w -40 -safe-string -g -I otherlibs/configurator/src/.configurator.objs/byte -I otherlibs/configurator/src/.configurator.objs/native -I otherlibs/configurator/src/.configurator.objs/public_cmi -I /home/opam/.opam/4.04/lib/csexp -I /home/opam/.opam/4.04/lib/result -intf-suffix .ml -no-alias-deps -open Configurator__ -o otherlibs/configurator/src/.configurator.objs/native/configurator__V1.cmx -c -impl otherlibs/configurator/src/v1.ml)
# File "otherlibs/configurator/src/v1.ml", line 532, characters 13-27:
# Error: Unbound value Int.Map.update
```
Is that correct?